### PR TITLE
adding new static text and following renames

### PR DESCRIPTION
**Pull request by sriGanna**
This PR is to reflect updates to the photo-audio-haptics-handler. 
The following changes have been made:

- adding two new static text segments to inform the user of movement to/from regions
- renaming to follow schema changes

The updated handler has been tested on unicorn against the images on the Image demo page as well as the following images: 

- https://en.wikipedia.org/wiki/File:Canyon_no_Lago_de_Furnas.jpg
- https://raw.githubusercontent.com/Shared-Reality-Lab/auditory-haptics-graphics-DotPad/main/preprocessor_JSON/photos/1_outdoor_cycling_scene/outdoor_cycling_scene.jpg?token=GHSAT0AAAAAABLKCO6OO6OCBNKRQV4WL4HUYQ772HA

Please ensure you've followed the checklist and provide all the required information *before* requesting a review.

## Required Information

- [ ] Reference the issue this is meant to fix or describe it if none exists.
- [ ] Full description of changes made.

## Pull Request Checklist

- [ ] Coding standards are followed (e.g., [PEP8](https://pep8.org/))
- [ ] An entry exists for the container in `docker-compose.yml` or `build.yml`
- [ ] The Docker image builds
- [ ] The container runs correctly when sent inputs with the changes
- [ ] No models or other large files are part of these commits
- [ ] A description of the tests already run as part of this PR is present
- [ ] CI is set up under `.github/workflows` or is not applicable

### CODE REVIEW

File: handlers/photo-audio-haptics-handler/src/server.ts [hunk 1]
```diff
@@ -20,6 +20,8 @@ import fetch from "node-fetch";
 import { v4 as uuidv4 } from "uuid";
 import fs from "fs/promises";
 
+
```
Comment (JRegimbal) [inline handlers/photo-audio-haptics-handler/src/server.ts:23]: Unnecessary whitespace.

File: handlers/photo-audio-haptics-handler/src/server.ts [hunk 2]
```diff
@@ -126,15 +130,22 @@ app.post("/handler", async (req, res) => {
         const [ttsInfo, geometryInfo] = utils.generateObjDet(preObjDet, preGroupData);
         ttsData.push(...ttsInfo);
         objGeometryData.push(...geometryInfo);
+        
     }
-
+
```
Comment (JRegimbal) [inline handlers/photo-audio-haptics-handler/src/server.ts:135]: Extra tab.

File: handlers/photo-audio-haptics-handler/src/server.ts [hunk 3]
```diff
@@ -126,15 +130,22 @@ app.post("/handler", async (req, res) => {
...
-
+    
     // Concatenate adjacent text entries
     for (let i = 0; i < ttsData.length - 1; i++) {
         if (ttsData[i].type === "text" && ttsData[i + 1].type === "text") {
             ttsData[i].value += " " + ttsData[i + 1].value;
             ttsData.splice(i + 1, 1);
         }
     }
+    if (preSemSeg) {
```
Comment (JRegimbal) [inline handlers/photo-audio-haptics-handler/src/server.ts:143]: Since this is just added to the end, then you're adding functionality in the browser to skip to this part when necessary, right?

File: handlers/photo-audio-haptics-handler/src/server.ts [hunk 4]
```diff
@@ -203,24 +220,41 @@ app.post("/handler", async (req, res) => {
...
                         };
 
-                    if (preObjDet && preSemSeg)
+                    if (preObjDet && preSemSeg) {
                         entities[1 + segGeometryData.length] = {
                             ...entities[1 + segGeometryData.length],
                             centroid: [[]],
-                            contourPoints: [[]]
+                            contours: [[]],
+                            entityType: "staticObjDet"
```
Comment (JRegimbal) [inline handlers/photo-audio-haptics-handler/src/server.ts:232]: So these are individual names representing an a specific part of TTS? It works, but keep in mind this will make it difficult to scale up without modifying the browser and will heavily limit reusability.

**Comment by JRegimbal** [comment]
@sriGanna is this ready to be merged?

**Comment by sriGanna** [comment]
Yes,please :)
